utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -18,30 +17,29 @@ private const val COUNTRY_USER_PROPERTY = "COUNTRY"
 @Singleton
 class FirebaseEventLogger(
   private val firebaseAnalytics: FirebaseAnalytics,
-  private val eventBundleCreator: EventBundleCreator,
-  context: Context
+  private val eventBundleCreator: EventBundleCreator
 ) : EventLogger {
   private var bundle = Bundle()
-  private val connectivityManager: ConnectivityManager =
-    context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+
+  @Inject
+  lateinit var networkConnectionUtil: NetworkConnectionUtil
```

Comment:
This should be injected in the constructor, not as a field.

Reply:
Fixed